tests/models/test_classic_ml.py

```diff
@@ -20,7 +20,7 @@ def test_linear_regression_model(tmpdir):
     loader = DataLoader(SklearnDataset(X, y), batch_size=2)
 
     model = LinearRegression(input_dim=2, learning_rate=0.5)
-    trainer = pl.Trainer(max_epochs=300, default_root_dir=tmpdir, progress_bar_refresh_rate=0)
+    trainer = pl.Trainer(max_epochs=300, default_root_dir=tmpdir, progress_bar_refresh_rate=0, logger=False)
```

Comment:
Let's turn off saving checkpoints here since we're not using `fast_dev_run=True`. [the docs](https://pytorch-lightning.readthedocs.io/en/stable/debugging.html#fast-dev-run)

```suggestion
    trainer = pl.Trainer(max_epochs=300, default_root_dir=tmpdir, progress_bar_refresh_rate=0, logger=False, checkpoint_callback=False)
```